components/button/button-subtle.js

```diff
@@ -1,7 +1,9 @@
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { LitElement, html } from 'lit-element/lit-element.js';
 import { D2LButtonMixin } from './button-mixin.js';
-import { buttonStyles } from './button-styles.js';
+import { labelStyles } from '../typography/styles.js';
```

Comment:
The standard button implementation would be similar:
```
import { labelStyles } from '../typography/styles.js';
import { buttonSharedStyles } from './button-shared-styles.js';
import { buttonStyles } from './button-styles.js';
```

And the icon button implementation would be like:
```
import { buttonSharedStyles } from './button-shared-styles.js';
import { buttonIconStyles } from './button-icon-styles.js';
```